clouds/aws/aws_roles/aws_roles.py

```diff
@@ -0,0 +1,66 @@
+import botocore.errorfactory
+from aws.aws_clients import AWSClient
+from botocore.exceptions import BotoCoreError
+from simple_logger.logger import get_logger
+
+LOGGER = get_logger(name=__name__)
+
+
+def get_iam_client():
+    return AWSClient(service_name="iam").client
+
+
+def create_role_policy(role_name, policy_name, policy_document_path):
+    """
+    Create the role policy with specific configuration.
```

Comment:
what does `with specific configuration.` mean?

Reply:
Required configurations such as the unique role name combined with policy name and policy doc with specified instructions. Delete?

Reply:
Required configurations such as the unique role name combined with policy name and policy doc with specified instructions. Delete?